src/compiler/diagnosticMessages.json

```diff
@@ -4270,6 +4270,10 @@
         "category": "Error",
         "code": 18005
     },
+    "Private names are not currently supported ": {

```

Comment:
Do error messages typically end with punctuation? I noticed the one below ends with a period.